README.rst

```diff
@@ -81,6 +84,18 @@ submodule update --init``. This will fetch the `Makd
 <https://github.com/sociomantic-tsunami/makd>`_ project in ``submodules/makd``.
 
 
+Submodules
+**********
+
+============================= ===========
+Dependency                     Version
+============================= ===========
+makd                           v1.3.x
+libtangort-dmd-dev (for D1)    v1.6.0
+dmd-transitional   (for D2)    2.070.2.s9
```

Comment:
`libtango` and `dmd-transitional` are not submodule dependencies.

This should probably be in the `Packages` section ... ?  I would suggest explicitly noting the repository from where these packages can be installed.  We should probably also reference the `dmd1` compiler.

Reply:
Aha, good point. At this juncture, I would defer to the knowledge of someone else ( @mihails-strasuns-sociomantic ? @leandro-lucarella-sociomantic ?) about what exactly would be the best thing to note here.

Reply:
BTW, do I recall right that `Makd` relies on `rdmd` being present?  We might want to mention that as well.

All things considered, we might want a separate `Compiler/Runtime` dependencies section to cover compilers and runtime/standard libraries.

Reply:
The other option might be to have a standardized piece of documentation for our D libraries, describing the compilers and runtimes needed, which we can link to from individual project READMEs.

Reply:
Probably we should point the users to look at the supported versions based on the test matrix, but it's true that normally we probably also support older versions too (by coincidence). Maybe is enough to say exactly that.